docs/articles/voice/azure-speech.md

```diff
@@ -0,0 +1,134 @@
+# DSharpPlus + Azure Speech Services
+
+## 1. Introduction
+
+With a little bit of effort, it is entirely possible to leverage Microsoft Azure's Cognitive Services for your Discord bot.  Of particular interest are the speech services, which will allow you to do [text-to-speech](https://docs.microsoft.com/en-us/azure/cognitive-services/speech-service/text-to-speech) (the focus of this article) and [speech-to-text](https://docs.microsoft.com/en-us/azure/cognitive-services/speech-service/speech-to-text).  This article will show you how to create a command to have your bot say something back to you, like Simon says.  Once you've got that under your belt, you can make modifications to have your bot speak to you based on whatever trigger and content your heart desires.  If you are feeling ambitious, you can even have your bot respond to voice messages using speech-to-text.  Just parse the returned text as if it were a regular Channel message or pass it through your Interactivity module for some real fun.
+
+## 2. Prerequisites
+
+* DSharpPlus >= 4.0
+* DSharpPlus.VoiceNext
+  * This article assumes that you've already followed the [VoiceNext](https://dsharpplus.emzi0767.com/articles/vnext_setup.html) article and have a bot that works and that can play music or some other audio.
```

Comment:
Do not use fully-qualified links, especially since that domain is getting nixed when 4.0 goes live. You can use absolute links (i.e. `/articles/vnext_setup.html`).